src/components/Account/Account.test.tsx

```diff
@@ -6,12 +6,13 @@ import customer from '#test/fixtures/customer.json';
 import { useAccountStore } from '#src/stores/AccountStore';
 import { renderWithRouter } from '#test/testUtils';
 import type { Consent } from '#types/account';
+import type { ConsentFieldVariants } from '#src/services/inplayer.account.service';
```

Comment:
We should probably think of some other way to handle the enums, so we don't have imports from services into the components.
The not recognising exported enums from other files may be a vite problem, but if it's not doable, then maybe switch to using type union or object/array variables.

Reply:
I hate replacing enums with union types, but it unfortunately seems unavoidable.